go/libraries/doltcore/sqle/remotesrv.go

```diff
@@ -131,7 +131,7 @@ func (si SqlContextServerInterceptor) Stream() grpc.StreamServerInterceptor {
 		defer sql.SessionEnd(sqlCtx.Session)
 		sql.SessionCommandBegin(sqlCtx.Session)
 		defer sql.SessionCommandEnd(sqlCtx.Session)
-		newCtx := context.WithValue(ss.Context(), sqlContextInterceptorKey{}, sqlCtx)
+		newCtx := context.WithValue(sqlCtx.Context, sqlContextInterceptorKey{}, sqlCtx)
```

Comment:
Since sql.Context embeds context.Context, you can just pass it directly here, no need to name the embedded field.